packages/experimental/Badge/src/BadgeColorTokens.ts

```diff
@@ -1,145 +1,493 @@
+import { ColorValue } from 'react-native';
 import { Theme, TokenSettings } from '@fluentui-react-native/framework';
 import { globalTokens } from '@fluentui-react-native/theme-tokens';
 import { BadgeTokens } from './Badge.types';
 
 export const defaultBadgeColorTokens: TokenSettings<BadgeTokens> = (t: Theme) =>
   ({
     filled: {
-      backgroundColor: t.colors.brandedBackground,
-      color: t.colors.neutralForegroundInverted,
-      iconColor: t.colors.neutralForegroundInverted,
+      ...getFilledColorProps(
+        {
+          backgroundColor: t.colors.brandBackgroundStatic,
+        },
+        t,
+      ),
       borderColor: 'transparent',
     },
     outline: {
+      ...getOutlineColorProps({ color: t.colors.brandForeground1, colorDark: '#62ABF5' }, t),
       backgroundColor: t.colors.transparentBackground,
-      color: t.colors.brandForeground1,
-      borderColor: t.colors.brandForeground1,
-      iconColor: t.colors.brandForeground1,
     },
     tint: {
-      backgroundColor: t.colors.brandBackground2,
-      color: t.colors.brandForeground1,
-      borderColor: t.colors.brandStroke2,
-      iconColor: t.colors.brandForeground1,
+      ...getTintColorProps(
+        {
+          backgroundColor: globalTokens.color.brand.tint60,
+          color: t.colors.brandForeground1,
+          borderColor: t.colors.brandStroke2,
+          backgroundColorDark: globalTokens.color.outlook.shade40,
+          colorDark: '#77B7F7',
+          borderColorDark: globalTokens.color.outlook.shade40,
+        },
+        t,
+      ),
+    },
+    ghost: {
+      backgroundColor: t.colors.transparentBackground,
+      borderColor: 'transparent',
+      ...getGhostColorProps(
+        {
+          color: t.colors.brandForeground1,
+          colorDark: '#62ABF5',
```

Comment:
>colorDark: '#62ABF5', [](http://example.com/codeflow?start=10&length=21)

You can just use brandForeground1.

All brand-based badges should use alias tokens. Otherwise they won't change their color to match the host app's brand.